input/school_aliases/template.toml

```diff
@@ -0,0 +1,3 @@
+# replace the placeholder IDs below with school IDs so that they are replaced during the extract process
+RRS200 = "RRS100"
```

Comment:
seems reasonable, but this template file shouldn't need to be changed for the tests to run. I'd do all my developing using the new test `test_school_id_aliases`